components/Grid/grid.html

```diff
@@ -0,0 +1,237 @@
+                        </li>
+                        <li>
+                            <a href="/components/Modal/modal.html">
+                                Modal
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/components/Grid/grid.html">
+                                Grid
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="doc-content">
+
+                <section class="Grid">
+
+                    <h4 class="mb-3 text-primary-color">Grids</h4>
+                    <p class="mb-3 "> CSS Grid is a multi-dimension layout module, which means that it has columns and
+                        rows
+                    </p>
+                    <h6 class="mt-5 mb-3 text-primary-color"> Grid Variant: </h6>
+                    <p class="mb-3">Grid 50-50 layout</p>
+                    <div class="component-display">
+
+                        <div class="grid-50-50">
+                            <div>
+                                <img class="img-responsive"
+                                    src="https://m.media-amazon.com/images/I/71Sdj04A8dS._SL1500_.jpg"
+                                    alt="smart 4k television">
+                            </div>
+
+
+                            <div class="d-flex flex-center flex-col">
+                                <h3 class="mb-2">Great Sale!</h3>
+                                <h5 class="mb-2">1m 08cm (43") AUE60 Crystal 4K UHD Smart TV</h5>
+                                <button class="btn btn-outline-primary my-4">
+                                    Buy now!
+                                </button>
+                            </div>
+
+                        </div>
+
+                    </div>
+                    <div class="code-snippet">
+                        <code>
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-50-50%2522%253E%250A%2509%253Cdiv%253E%2520%253Cimg%2520class%253D%2522img-responsive%2522%2520src%253D%2522%252Fassets%252Ftv.webp%2522%2520alt%253D%2522smart%25204k%2520television%2522%253E%2520%253C%252Fdiv%253E%250A%2509%253Cdiv%2520class%253D%2522d-flex%2520flex-center%2520flex-col%2522%253E%250A%2509%2509%253Ch3%2520class%253D%2522mb-2%2522%253EGreat%2520Sale%21%253C%252Fh3%253E%250A%2509%2509%253Cbutton%2520class%253D%2522btn%2520btn-outline-primary%2520my-4%2522%253E%2520Buy%2520now%21%2520%253C%252Fbutton%253E%250A%2509%253C%252Fdiv%253E"
+                                style="width: 100%; height: auto;	; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Please create a class instead of inline css

Reply:
Yes, i have to refactor this in every component.